src/style-spec/reference/v8.json

```diff
@@ -2490,6 +2490,42 @@
           }
         }
       },
+      "round": {
+        "doc": "Rounds the input to the nearest integer. Halflway values are rounded away from zero. E.g., `[\"round\", -1.5]` evaluates to -2.",
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.45.0"
+          }
+        }
+      },
+      "abs": {
+        "doc": "Returns the absolute value of the input.",
```

Comment:
when implementing I noticed abs on `-1.1` returns `1`, could we include in the documentation that abs is an integer type function/not floating point? (from a java background this was a bit confusing for me).